server/routes/prisonerSearch.ts

```diff
@@ -55,20 +57,50 @@ export default function prisonerSearchRoutes(service: Services): Router {
 
       if (form && !form.hasErrors) {
         const page = form.fields.page.value
+        const scope = form.fields.scope.value
         const searchTerms = form.fields.q.value
         const sort = form.fields.sort.value
         const order = form.fields.order.value
 
-        const response = await offenderSearchClient.searchInPrison(prisonId, searchTerms, page - 1, sort, order)
+        let response: OffenderSearchResults
+        const globalSearch = hasGlobalSearch && scope === 'global'
+        if (globalSearch) {
+          const filters: Parameters<OffenderSearchClient['searchGlobally']>[0] = {
+            location: hasGlobalSearchAndInactiveBookings ? 'ALL' : 'IN',
+            includeAliases: true,
+          }
+          if (/\d/.test(searchTerms)) {
+            // DPS global search assumes the whole query is a prisoner identifier if it contains numbers
+            filters.prisonerIdentifier = searchTerms.toUpperCase()
+          } else {
+            // DPS global search assumes that up to 2 words are entered, being the last and first name in that order
+            const [lastName, firstName] = searchTerms.split(' ')
```

Comment:
Interesting quirk, so if someone search "FIRST LAST" may not return anything because it's not in the right order?

Also interesting that if it contains a digit it assumes is the prisoner number 👍

Reply:
yeah “FIRST LAST” will return the wrong results 🤭 

if the [search terms contain _any_ numbers](https://github.com/ministryofjustice/digital-prison-services/blob/c87246bb2714642b87599d7eee7236519c0215d4/backend/utils.ts#L204):
https://github.com/ministryofjustice/digital-prison-services/blob/c87246bb2714642b87599d7eee7236519c0215d4/backend/controllers/globalSearch.ts#L30

otherwise:
https://github.com/ministryofjustice/digital-prison-services/blob/c87246bb2714642b87599d7eee7236519c0215d4/backend/controllers/globalSearch.ts#L44-L49